### DISCUSSION TITLE
Shell scripts for administration

### DISCUSSION THREAD

**Discussion by m-overlund** [OpenMage/magento-lts#3554] [Ideas]
I've pieced together a couple of (far from perfect) shell scripts for daily administration tasks, missing in OpenMage - which most shop owners deals with:
- Attaching previously made guest orders to newly registrered customer
- Changing email for an order

https://github.com/m-overlund/magento1-phpcli-tools

Would maintainers support such tools in the core?

**Comment by fballiano** [comment]
IMHO those scripts would be perfect as part of n98-magerun 😊

**Reply by m-overlund** [reply]
Yeah, that would be nice - I'm not skilled enough to submit a PR for n98-magerun - however i will suggest it implemented.

**Comment by addison74** [comment]
Either they can be included in the /shell, or they can be added to another directory, for example /actions in a separate repository.

An action would be to delete an order with a certain number. There are extensions, but I don't know how good they are after all this time.

**Reply by m-overlund** [reply]
sounds like a plan - will submit PR
